.travis.yml

```diff
@@ -5,9 +5,20 @@ dist: bionic
 go:
   - "1.12.1"
 
-before_install:
-  - GO111MODULE=off go get github.com/mattn/goveralls
+
+os:
+  - linux
+  - windows
 
 script:
-  - go test -covermode=count -coverprofile=profile.cov ./pkg/...
-  - $GOPATH/bin/goveralls -coverprofile=profile.cov -service=travis-ci
+  - |
+    if [[ "${TRAVIS_OS_NAME}" == "linux" ]]; then
+      go test -covermode=count -coverprofile=profile.cov ./pkg/...
+      GO111MODULE=off go get github.com/mattn/goveralls
+      $GOPATH/bin/goveralls -coverprofile=profile.cov -service=travis-ci
+    elif [[ "${TRAVIS_OS_NAME}" == "windows" ]]; then
+      echo "$PROW_AAD_CLIENT_SECRET" | docker login -u "$PROW_AAD_CLIENT_ID" --password-stdin "$REGISTRY"
```

Comment:
The environment variables are not set in travis-ci. So login failed here. 

Maybe there are some problems with this azure sp. We need to discuss them with Andy @andyzhangx .

Reply:
We cannot store sp in travis-ci any more, it would leak secret.